actions-server/src/handlers/joinGame.ts

```diff
@@ -1,46 +1,41 @@
-const fetch = require("node-fetch")
-const jwt = require("jsonwebtoken")
-
-const INSERT_OPERATION = `
-mutation InsertPlayerForGame($gameId: uuid!, $clientUuid: uuid!) {
-  insert_players_one(
-    object: { game_id: $gameId, client_uuid: $clientUuid }
-  ) {
-    id
-  }
-}
-`
-
-const LOOKUP_OPERATION = `
-query LookupPlayerForGame($gameId: uuid!, $clientUuid: uuid!) {
-  players(where: {game_id: {_eq: $gameId}, client_uuid: {_eq: $clientUuid}}) {
-    id
-  }
-}
-`
+import { Request, Response } from "express"
+import { DocumentNode, print as gqlToString } from "graphql"
+import jwt from "jsonwebtoken"
+import fetch from "node-fetch"
+import {
+  InsertPlayerForGame,
+  InsertPlayerForGameMutationVariables,
+  LookupPlayerForGame,
+  LookupPlayerForGameQueryVariables,
+} from "../generated/graphql"
```

Comment:
haven't yet figured out how to make this non relative. was able to for app/ ... 

I thought it was related to the tsconfig setting "baseUrl"

Reply:
This seems to work for me:

```suggestion
  InsertPlayerForGame,
  InsertPlayerForGameMutationVariables,
  LookupPlayerForGame,
  LookupPlayerForGameQueryVariables,
} from "generated/graphql"
```

Reply:
Nice, weird I tried this.. maybe just needed to refresh my local..

Reply:
weird.. cannot seem to get past it, will try again tmrw.

```
Error: Cannot find module 'generated/graphql'
Require stack:
- /Users/avinashmoondra/Desktop/fishbowl/actions-server/build/handlers/joinGame.js
- /Users/avinashmoondra/Desktop/fishbowl/actions-server/build/server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:976:15)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/avinashmoondra/Desktop/fishbowl/actions-server/build/handlers/joinGame.js:18:19)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1036:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/avinashmoondra/Desktop/fishbowl/actions-server/build/handlers/joinGame.js',
    '/Users/avinashmoondra/Desktop/fishbowl/actions-server/build/server.js'
  ]
}
```